cli/ballerina-cli/src/main/java/io/ballerina/cli/task/RunTestsTask.java

```diff
@@ -453,6 +461,112 @@ private int runTestSuit(Target target, Package currentPackage, JBallerinaBackend
         return proc.waitFor();
     }
 
+    private int runTestSuiteWithNativeImage(Package currentPackage, JBallerinaBackend jBallerinaBackend, Target target)
+            throws IOException, InterruptedException {
+        String packageName = currentPackage.packageName().toString();
+        String classPath = getNewClassPath(jBallerinaBackend, currentPackage);
+        String jacocoAgentJarPath = "";
+
+        if (coverage) {
+            // Generate the exec in a separate process
+            List<String> cmdArgs1 = new ArrayList<>();
```

Comment:
Let's use a meaningful name instead of `cmdArgs1`

Reply:
Changed to `execArgs`